package.json

```diff
@@ -1,35 +1,37 @@
 {
-  "name": "deprem-io-backend",
```

Comment:
bu dosyada yapilan degisikleri kaldirabilir misin? indentation'in degismemesi lazim.